doc/release/release_notes_v1_2.md

```diff
@@ -316,7 +316,10 @@ For a discussion of issues and options, contact us via the
 [mailing list](https://daos.groups.io/g/daos).
 
 ### Limited bandwidth to SCM with some update/write operations
```

Comment:
this title needs changing if the bandwidth limitation no longer applies (the text related to the bandwidth limitation has been removed below)

Reply:
How about - 
Nested Subdirectories are not Supported as a Valid Mount Point?

Reply:
sounds great